deny.toml

```diff
@@ -74,6 +74,7 @@ unlicensed = "deny"
 allow = [
   "MIT",
   "Apache-2.0",
+  "Apache-2.0 WITH LLVM-exception",
```

Comment:
This is due to transitive dependencies around cranelift-* licenses. It is bit cryptic to read, but peeking https://bugzilla.mozilla.org/show_bug.cgi?id=1476798 I think it's reasonably safe to allow.